src/Tags/Increment.php

```diff
@@ -2,10 +2,17 @@
 
 namespace Statamic\Tags;
 
-class Increment extends Tags
+use Statamic\View\State\ResetsState;
+
+class Increment extends Tags implements ResetsState
 {
     protected static $arr = [];
 
+    public function resetState()
```

Comment:
```suggestion
    public static function resetStaticState()
```

See other comment for explanation.